Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -86,11 +86,16 @@ public void setWordCount(String url, int wordCount) {
 
   public void insertScriptedData(String url) {
     SQLiteDatabase dataBase = getWritableDatabase();
-    try {
-      dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
-      dataBase.close();
-    } catch (Exception exception) {
-      dataBase.close();
+    dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
+    dataBase.close();
+  }
+
+  public void insertScriptedDataWithCheckDuplication(String url) {
```

Comment:
I create temporarily method `insertScriptedDataWithCheckDuplication`
please using this method when insert url with no duplication.

but I think 

> scan new feedly url list -> check duplication url with database url list and insert.

is more good performance 
@leesera